loki/backend/cufgen.py

```diff
@@ -0,0 +1,60 @@
+from loki.backend.fgen import FortranCodegen
+
+__all__ = ['cufgen', 'CudaFortranCodegen']
+
+
+class CudaFortranCodegen(FortranCodegen):
+    """
+    Tree visitor that extends `FortranCodegen` with Cuda Fortran (CUF) language variations.
+    """
+
+    def __init__(self, depth=0, indent='  ', linewidth=90, conservative=True):
+        super().__init__(depth=depth, indent=indent, linewidth=linewidth, conservative=conservative)
+
+    def visit_CallStatement(self, o, **kwargs):
+        """
+        Format call statement as
+          CALL(<chevron>) <name>(<args>)
+          with the chevron as launch configuration for device offloading,
+          resulting in something like
+          call kernel<<<grid,block[,bytes][,streamid]>>>(arg1,arg2,...)
+        """
+        pragma = self.visit(o.pragma, **kwargs)
+        name = self.visit(o.name, **kwargs)
+        args = self.visit_all(o.arguments, **kwargs)
```

Comment:
Does CUF not allow keyword arguments?

If it does allow them, these should be printed, too. Otherwise, I suggest to add a `assert not o.kwarguments`.